src/services/api/src/app.py

```diff
@@ -158,6 +158,18 @@ async def ping():
     return "pong"
 
 
+@app.get("/state", status_code=200)
+async def state():
+    """Endpoint to get the state of the dispatcher."""
+    if os.environ.get("DEV_MODE", "false") == "true":
```

Comment:
Currently accessible only in dev mode. Unsure whether to just leave this as a public endpoint, or later secure via some form of RBAC